github/repos_hooks_deliveries.go

```diff
@@ -0,0 +1,205 @@
+// Copyright 2021 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+)
+
+// HookDelivery represents the data that is received from GitHub's Webhook Delivery API
+//
+// GitHub API docs:
+// - https://docs.github.com/en/rest/reference/repos#list-deliveries-for-a-repository-webhook
+// - https://docs.github.com/en/rest/reference/repos#get-a-delivery-for-a-repository-webhook
+type HookDelivery struct {
+	ID             *int64     `json:"id"`
+	GUID           *string    `json:"guid"`
+	DeliveredAt    *time.Time `json:"delivered_at"`
+	Redelivery     *bool      `json:"redelivery"`
+	Duration       *float64   `json:"duration"`
```

Comment:
Shouldn't this be `*int64` ?  Where is this documented?
Neither of the URLs above have a `duration` field on their pages.

Reply:
It isn't documented but looks like the below example. I've grabbed this by calling the list deliveries API on my own repo:

```
  {
    "id": 12076761161,
    "guid": "5a73eedc-d706-11eb-9a56-eb84e3dea363",
    "delivered_at": "2021-06-27T05:13:05Z",
    "redelivery": false,
    "duration": 0.29,
    "status": "Invalid HTTP Response: 530",
    "status_code": 530,
    "event": "push",
    "action": null,
    "installation_id": null,
    "repository_id": 366236975
  },
```

Reply:
Awesome. Thank you. Please ignore this comment of mine.